### PR TITLE
docs: remove NAT rule when removing bridge

### PR DESCRIPTION
**Pull request by imreFitos**
Signed-off-by: imre Fitos <imre.fitos+github@gmail.com>

### CODE REVIEW

**Comment by MalteJ** [comment]
Do you remove the nat tables and afterwards check that they are still existing?
I think you'd need to start the docker daemon first with iptables enabled.

**Comment by imreFitos** [comment]
I remove the nat tables so docker can create new correct ones for the new bridge. Docker won't update the nat tables if it finds an existing POSTROUTING rule. 

The check command just shows if the correct entries are in place.

**Comment by jamtur01** [comment]
LGTM

**Comment by jamtur01** [comment]
Ping @SvenDowideit @fredlf

**Comment by MalteJ** [comment]
> I remove the nat tables so docker can create new correct ones for the new bridge. Docker won't update the nat tables if it finds an existing POSTROUTING rule.

hmm, I don't get it.
You flush iptables at line 690. After that at line 714 you do `iptables -t nat -L -n`. Imho this output should still be empty. Then at line 723 you start the docker daemon. Only now the iptables should show `MASQUERADE  all  --  192.168.5.0/24      0.0.0.0/0`.

File: docs/sources/articles/networking.md [hunk 1]
```diff
@@ -708,6 +709,14 @@ illustrate the technique.
         inet 192.168.5.1/24 scope global bridge0
            valid_lft forever preferred_lft forever
 
+    # Confirming outgoing NAT masquerade is setup
```
Comment (fredlf) [inline docs/sources/articles/networking.md]: set up

**Comment by fredlf** [comment]
LGTM after typo is fixed

**Comment by imreFitos** [comment]
You are right, the second addition should be after the "Tell Docker about it and restart" section. Updating.

**Comment by MalteJ** [comment]
@imreFitos great :smiley:

**Comment by imreFitos** [comment]
fixed typo

**Comment by jamtur01** [comment]
LGTM

**Comment by jamtur01** [comment]
Build broken? Perhaps re-run it.

**Comment by jfrazelle** [comment]
I clicked the button to rebuild

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**